.circleci/config.yml

```diff
@@ -12,6 +12,14 @@ references:
       DOCKER_CLIENT_VERSION: "17.09.0-ce"
       DOCKERHUB_REPO: aeternity/epoch
 
+
+  machine_config: &machine_config
+    machine: true
```

Comment:
I see this is a VM https://circleci.com/docs/2.0/executor-types/#using-machine
Please ensure the final commit message has a note on why needed.